controllers/user.controller.js

```diff
@@ -8,71 +10,79 @@ const {
   getPublicUserData,
   respondWithError,
   respondWithStatus,
-} = require("../services/utils");
+} = require('../services/utils');
 
-const bcrypt = require("bcrypt");
+const bcrypt = require('bcrypt');
 
-const { SERCURITY, ERRORS, CRUD_OPS } = require("../constants");
+const { SERCURITY, ERRORS, CRUD_OPS } = require('../constants');
 
 const register = async (req, res) => {
   try {
     const { name, email, password } = req.body;
 
     if (await userService.isUserExists(name, email)) {
-      throw new Error(ERRORS.NAME_OR_EMAIL_ERROR);
+      return res.status(HttpStatusCode.Conflict).send('user already exists');
     }
```

Comment:
I think we already have status codes in the constant folder

Reply:
this was not a good code in my opinion, there was some incorrect code over there and I prefer to use a ready library insted of writing one from scratch